style.css

```diff
@@ -22,3 +22,68 @@ body {
 }
 
 /* Add styles here! */
+
+/* Typography */
+h1 {
+  font-size: 2rem;
+  margin-block: 32px;
+}
+
+h2 {
+  font-size: 1.5rem;
+  margin-block: 24px;
+}
+
+p {
+  font-size: 1rem;
+  line-height: 1.5;
+  margin-block: 16px;
+}
+
+/* Layout */
+header {
+  padding-block: 64px;
+}
+
+main {
+  padding-block: 64px;
+  border-top: 8px solid var(--color-gray-medium);
+  background-color: var(--color-gray-light);
+}
+
+.max-width-wrapper {
+  max-width: 768px;
+  margin-inline: auto;
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💡 thought:** ‏These logical property shortcuts are new and nice no?
Before we should have written
```
margin-left: auto;
margin-right: auto;
```